docs/core/project-sdk/msbuild-props.md

```diff
@@ -1421,6 +1421,116 @@ The `RunWorkingDirectory` property defines the working directory for the applica
 </PropertyGroup>
 ```
 
+## Test project&ndash;related properties
+
+The following MSBuild properties are documented in this section:
+
+- [IsTestProject](#istestproject)
+- [IsTestingPlatformApplication](#istestingplatformapplication)
+- [Enable\[NugetPackageNameWithoutDots\]](#enablenugetpackagenamewithoutdots)
+- [EnableAspireTesting](#enableaspiretesting)
+- [EnablePlaywright](#enableplaywright)
+- [EnableMSTestRunner](#enablemstestrunner)
+- [EnableNUnitRunner](#enablenunitrunner)
+- [GenerateTestingPlatformEntryPoint](#generatetestingplatformentrypoint)
+- [TestingPlatformCaptureOutput](#testingplatformcaptureoutput)
+- [TestingPlatformCommandLineArguments](#testingplatformcommandlinearguments)
+- [TestingPlatformDotnetTestSupport](#testingplatformdotnettestsupport)
+- [TestingPlatformShowTestsFailure](#testingplatformshowtestsfailure)
+- [TestingExtensionsProfile](#testingextensionsprofile)
+- [UseVSTest](#usevstest)
+
```

Comment:
I would also add `DisableTestingPlatformServerCapability` that removes the project capability used by VS and VS Code Test Explorers to know which platform protocol to use (VSTest or Microsoft.Testing.Platform).

Note that at the moment, the Microsoft.Testing.Platform is not yet avaiable publicly but will be soon.

Reply:
Tracking in #42544